src/helpers/ci.ts

```diff
@@ -486,8 +486,10 @@ export const getCISpanTags = (): SpanTags | undefined => {
 export const getCIMetadata = (): Metadata | undefined => {
   const tags = {
     ...getCISpanTags(),
+    ...getUserCIMetadata(),
     ...getUserGitMetadata(),
```

Comment:
there's a bit of confusion with `get...Metadata` and `get...SpanTags`. Sometimes `get...Metadata` returns a plain dictionary string to string (like `getUserCIMetadata` and `getUserGitMetadata`) and sometimes it returns a "complex" dictionary, like `getCIMetadata`. I'm not saying it should be fixed in this PR, just pointing it out. Maybe renaming every function that returns a plain string:string dictionary to `get...SpanTags` might be a good idea.

Reply:
I completely agree, when I tried to add  `getUserCIMetadata` it was already confusing.

Reply:
https://github.com/DataDog/datadog-ci/pull/460